### PR TITLE
Add pricing note to topline docs page directly

### PR DESCRIPTION
**Pull request by Joelkw**
This was already on the linked page but moving it up for visibility

### CODE REVIEW

File: doc/code_insights/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>
```
Comment (AlicjaSuska) [inline doc/code_insights/index.md:2]:
```suggestion
Code Insights are the analytics tool that tells you things about your codebase at a high level based on how your code changes over time.

Code Insights are based on our universal code search, making them incredibly accurate and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
```

Comment (AlicjaSuska) [reply doc/code_insights/index.md:2]: I'd add some intro text here aligned with the popup we're working on.

File: doc/code_insights/index.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
-Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md).
+> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, it may cost an additional amount.
```
Comment (AlicjaSuska) [inline doc/code_insights/index.md:3]:
```suggestion
> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, to continue using Code Insights may require a separate paid plan.
```

Comment (AlicjaSuska) [reply doc/code_insights/index.md:3]: @Joelkw aligning with the popup copy, it sounds more thoughtful imo

File: doc/code_insights/index.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>
 
-Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md).
+> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, it may cost an additional amount. 
 
-If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).
+Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md). If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).
 
-> NOTE: Code insights docs are actively in progress.
+Code insights docs are actively in progress during the beta.
```
Comment (AlicjaSuska) [inline doc/code_insights/index.md:8]:
```suggestion
It means that we're still polishing Insights and you might find bugs while we’re in beta. Please, do [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) to help us make Code Insights better.
```

Comment (AlicjaSuska) [reply doc/code_insights/index.md:8]: @Joelkw adding info about feedback, we can link this place from the popup.

File: doc/code_insights/index.md (hunk 1 above)
Comment (Joelkw) [reply doc/code_insights/index.md:2]: Sure! Definitely we want more context on this page/more of a "designed" feel to the page as we add more docs, similar to other top-level docs pages; this is a good start.

File: doc/code_insights/index.md (hunk 2 above)
Comment (Joelkw) [reply doc/code_insights/index.md:5]: Sounds good!

File: doc/code_insights/index.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
 # Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>
+Code Insights are the analytics tool that tells you things about your codebase at a high level based on how your code changes over time.
```
Comment (Joelkw) [inline doc/code_insights/index.md:2]:
```suggestion
Code Insights tells you things about your codebase at a high level, based on both how your code changes over time and its current state.
```

File: doc/code_insights/index.md [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
+Code Insights are based on our universal code search, making them incredibly accurate and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
+> NOTE: While in Beta, Code Insights is free for enterprise customers. Once Code Insights is generally available and no sooner than January 1, 2022, to continue using Code Insights may require a separate paid plan.
 
-If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).
-
-> NOTE: Code insights docs are actively in progress.
+Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md). If you're a customer who wants early access or needs support, please [send us an email](mailto:feedback@sourcegraph.com).
 
+Code insights docs are actively in progress during the beta.
+It means that we're still polishing Insights and you might find bugs while we’re in beta. Please, do [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) to help us make Code Insights better.
```
Comment (Joelkw) [inline doc/code_insights/index.md:10]:
```suggestion
We're still polishing Code Insights and you might find bugs while we’re in beta. Please [share any bugs 🐛 or feedback](mailto:feedback@sourcegraph.com) you have to help us make Code Insights better.
```

File: doc/code_insights/index.md [hunk 6]
```diff
@@ -1,11 +1,13 @@
 # Code Insights <a href="../admin/beta_and_experimental_features"><sup><span class="badge badge-beta">Beta</span></sup></a>
+Code Insights tells you things about your codebase at a high level, based on both how your code changes over time and its current state.
 
-Code Insights are currently in Private [Beta](../admin/beta_and_experimental_features.md).
+Code Insights are based on our universal code search, making them incredibly accurate and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
```
Comment (Joelkw) [inline doc/code_insights/index.md:4]:
```suggestion
Code Insights is based on our universal code search, making it incredibly precise and extremely configurable. Track anything that can be expressed with Sourcegraph search query: migrations, package use, version adoption, code smells, codebase size and much more, across 1,000s of repositories.
```

**Review by AlicjaSuska — APPROVED** [review]

**Pull request merged**